### PR TITLE
Adding 2 scrapers

### PR DESCRIPTION
**Pull request by CaptainStabs**
Dunwoody still needs a readme, Norcross readme should probably be made more user friendly.

### CODE REVIEW

File: USA/GA/Norcross/NorcrossGA_Scraper.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	r_data = response['data']
+	total = response['total']
+	base_url = 'https://www.norcrossga.net'
+	for p in range(total):
+		doc_url = r_data[p]['URL']
+		output_url = base_url + doc_url
+		
+		with open('urls.txt','a') as output:
+			output.write(output_url + '\n')
+output.close()
```
Comment (RasmusFonseca) [inline USA/GA/Norcross/NorcrossGA_Scraper.py:18]: `with` should automatically close output

File: USA/GA/DunwoodyPD/Dunwoody_scraper.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+while start_year <= int(max_year): 
+	end_month = int(start_month) + 2
+
+	# Convert the int value of the month to mm format
+	if len(str(start_month)) == 1:
+		start_month = str(start_month).zfill(2)
+	if len(str(end_month)) == 1:
+		end_month = str(end_month).zfill(2)
+
+	# Compensate for october with end_year change
```
Comment (RasmusFonseca) [inline USA/GA/DunwoodyPD/Dunwoody_scraper.py:51]: Why does october need to be compensated for?

Comment (CaptainStabs) [reply USA/GA/DunwoodyPD/Dunwoody_scraper.py:51]: October is the end of the year in the dataset as it updates every 2 months, without it the year would not progress and the months would not reset to `01`

**Comment by CaptainStabs** [comment]
Newer version supersedes this PR

**Pull request closed without merging**